avtozip/store/templates/store/main.html

```diff
@@ -1 +1,3 @@
 {% extends "main.html" %}
+
+{% block content %}== Content from store/main.html =={% endblock %}
```

Comment:
Do we really need it? In y opinion Content should be substituted completely. Need to discuss.